app/cli/src/types/enums.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: This will be clearer once the Plugin Interface
```

Comment:
We might need to move this into a package instead having it in the cli because these enums will probably useful for a Node-RED flow. However we can do that as a separate refactor

Reply:
Im thinking of doing the same for handlers.